core/client-internal/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -270,6 +272,7 @@ public boolean hasLocalWorker() {
    * Re-initializes the {@link BlockStoreContext}. This method should only be used in
    * {@link ClientContext}.
    */
+  @SuppressFBWarnings
```

Comment:
I think it's fine to add the suppression here - I'm removing this method anyway with https://tachyon.atlassian.net/browse/TACHYON-1623